src/mono/sample/iOS/Makefile

```diff
@@ -15,24 +19,51 @@ runtimepack:
 	../../../../build.sh Mono+Libs -os iOSSimulator -arch $(MONO_ARCH) -c $(MONO_CONFIG)
 
 run: clean appbuilder
-	$(DOTNET) publish -c $(MONO_CONFIG) /p:TargetArchitecture=$(MONO_ARCH) \
-	/p:UseLLVM=$(USE_LLVM) /p:ForceAOT=$(AOT)
+	$(DOTNET) publish \
+	-c $(MONO_CONFIG) \
+	/p:TargetArchitecture=$(MONO_ARCH) \
+	/p:UseLLVM=$(USE_LLVM) \
+	/p:ForceAOT=$(AOT) \
+	'/p:RuntimeComponents="$(RUNTIME_COMPONENTS)"' \
```

Comment:
Why this format
`'/p:RuntimeComponents="$(RUNTIME_COMPONENTS)"' \`
instead of like the previous formats
`/p:RuntimeComponents=$(RUNTIME_COMPONENTS) \`

Reply:
Since it includes , in the value it needs to be quoted or you will get msbuild argument parsing errors, just putting "" around value is not enough so complete property needs to be quoted as well in order to correctly go through shell parsing rules, problem is described here:

https://github.com/dotnet/msbuild/issues/2999

and there are various workarounds to the problem, one of them is used here.

Reply:
Oh, good to know, thanks for the explanation :)